mappers/tableMapper.js

```diff
@@ -0,0 +1,50 @@
+Table = require('../models/table');
+
+function createTable(tableNumber,tableLabels,size,seatLabels,eventId,callback){
+	var newTable = new Table({tableNumber:tableNumber, tableLabels:tableLabels, size:size,seatLabels:seatLabels,eventId:eventId});
+	newTable.save(function (err,product) {
+		return callback(err,product);
+	});
+}
+
+function findTablesByEventId(eventId,callback){		
+	Table.find({eventId:eventId}, function(err,res){
+		if(err){
+			return callback(err);
+		}
+		else{
+			return callback(err,res);
+		}
+	});	
+}
+
+function editTable(eventId,tableNumber,tableObj,callback){
+	Table.findOne({eventId:eventId,tableNumber:tableNumber}, function(err,res){
```

Comment:
This works fine but just for the future there's a mongoose function to exactly this: findOneAndUpdate() - avoids boilerplate of resetting the fields to save